openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx

```diff
@@ -588,141 +692,17 @@ const TagsPage = () => {
-                                    data-testid="usage-count">
-                                    Not used
-                                  </span>
-                                )}
-                              </div>
-                            </td>
-                            <td className="tableBody-cell">
-                              <div className="tw-text-center">
-                                <button
-                                  className="link-text"
-                                  data-testid="delete-tag"
-                                  disabled={!categoryPermissions.EditAll}
-                                  onClick={() =>
-                                    setDeleteTags({
-                                      data: {
-                                        id: tag.id as string,
-                                        name: tag.name,
-                                        categoryName: currentCategory.name,
-                                        isCategory: false,
-                                        status: 'waiting',
-                                      },
-                                      state: true,
-                                    })
-                                  }>
-                                  {deleteTags.data?.id === tag.id ? (
-                                    deleteTags.data?.status === 'success' ? (
-                                      <FontAwesomeIcon icon="check" />
-                                    ) : (
-                                      <Loader size="small" type="default" />
-                                    )
-                                  ) : (
-                                    <SVGIcons
-                                      alt="delete"
-                                      icon="icon-delete"
-                                      title="Delete"
-                                      width="16px"
-                                    />
-                                  )}
-                                </button>
-                              </div>
-                            </td>
-                          </tr>
-                        );
-                      }
-                    )
-                  ) : (
-                    <tr className="tableBody-row">
-                      <td className="tableBody-cell tw-text-center" colSpan={4}>
-                        No tags available.
-                      </td>
-                    </tr>
-                  )}
-                </tbody>
-              </table>
-            </div>
+            <Table
+              columns={tableColumn}
+              data-testid="table"
+              dataSource={
+                isEmpty(currentCategory?.children)
+                  ? []
+                  : (currentCategory?.children as TagClass[])
+              }
+              pagination={false}
+              size="small"
```

Comment:
`rowKey`

Reply:
You have added a `key` which has a different purpose. Please add `rowKey` as mentioned in the comment

Reply:
@chirag-madlani  updated!